packages/product/src/scripts/bin/run-seed.ts

```diff
@@ -15,5 +22,16 @@ export default (async () => {
     )
   }
 
+  const run = ModulesSdkUtils.buildSeedScript({
+    moduleName: Modules.PRODUCT,
+    models: ProductModels,
+    pathToMigrations: __dirname + "/../../migrations",
+    executorFn: async (manager, data) => {
```

Comment:
```suggestion
    seedFuncs: async (manager, data) => {
```

Reply:
seedHandler? it is a single handler wdyt?

Reply:
Sure!